installer/dmg/scripts/postinstall

```diff
@@ -0,0 +1,169 @@
+}
+
+#############################################################
+### Configure the newly installed drivers in odbcinst.ini ###
+#############################################################
+# Create the ODBC directory if it doesn't exist already
+if [[ ! -e $ODBC_PATH ]]; then
+    info "Creating $ODBC_PATH directory"
+    mkdir -p $ODBC_PATH
+elif [[ ! -d $ODBC_PATH ]]; then
+     err "$ODBC_PATH already exists but is not a directory"
+     exit 1
+fi
+
+#Update the odbcinst.ini with the latest driver path and mark it as installed
+if [[ ! -e "$INI" ]]; then
+    info ""$INI" does not exists, creating"
+    touch "$INI"
+    echo "[ODBC Drivers]" >> "$INI"
+    echo "$MDB = Installed" >> "$INI"
+    echo "" >> "$INI"
+    echo "[$MDB]" >> "$INI"
+    echo "Driver = $MDB_DRIVERS" >> "$INI"
+    echo "DriverUnicodeType = utf16" >> "$INI"
+    echo "" >> "$INI"
+    $CREATED_NEW_INI_FILE=1
+fi
+
+#If there was an existing odbcinst file, update the odbcinst.ini
+# with the latest driver path
+if [[ $CREATED_NEW_INI_FILE == 0 ]]; then
+    BACK_INI=""$INI".back"
+    info "Back up file $BACK_INI"
+    ODBC_DRIVERS_SECTION_EXISTS=0
+    CURRENT_SECTION="NONE"
+    PREVIOUS_SECTION="NONE"
+
+    # Make a copy of the existing ini file
+    cp "$INI" $BACK_INI
+    if [[ -e $BACK_INI ]]; then
+        # Clear the INI file
+        > "$INI"
+
+        # Read the backup odbcinst.ini.back file and recreate the new updated odbcinst.ini one
+        while IFS= read -r line
+        do
+            if [[ "$line" =~ ^\[.*\]$ ]]; then
+                PREVIOUS_SECTION=$CURRENT_SECTION
+                # Found a section
+                if grep -iqF "[ODBC Drivers]" <<< "$line"; then
+                    # Found the [ODBC Drivers] section 
+                    # where all drivers are listed
+                    ODBC_DRIVERS_SECTION_EXISTS=1
+                    CURRENT_SECTION="ODBC"
+                elif grep -iqF "[$MDB]" <<< "$line"; then
+                    # Found the [MongoDB Atlas SQL ODBC {version}] section
+                    CURRENT_SECTION="MongoDB Atlas SQL ODBC"
+                else
+                    CURRENT_SECTION="OTHER"
+                fi
+
+                # Found a new section right after [ODBC Drivers]
+                # Need to add the drivers lines in [ODBC Drivers] section
+                if [ $PREVIOUS_SECTION == "ODBC" ]; then
+                    flagDriversAsInstalled
+                    echo "" >> "$INI"
+                fi
+                echo "$line" >> "$INI"
+            else
+                if [ $CURRENT_SECTION == "ODBC" ]  && grep -iqF "$MDB" <<< "$line" ; then
+                    # Found the latest unicode MongoDB driver already configured
+                    # Make sure to mark it "Installed"
+                    echo "$MDB = Installed" >> "$INI"
+                    MDB_DRIVER_INSTALLED=1
+                elif [ $CURRENT_SECTION == "MongoDB Atlas SQL ODBC" ] && grep -iqF "Driver" <<< "$line" ; then
```

Comment:
```suggestion
                elif [[ $CURRENT_SECTION == "MongoDB Atlas SQL ODBC" ]] && grep -iqF "Driver" <<< "$line" ; then
```
This should have double brackets since the value has spaces otherwise it gets an `bash: [: too many arguments` error for the line.